src/main/java/edu/rit/honors/gyfp/api/folder/FolderApi.java

```diff
@@ -330,10 +330,12 @@ private void sendTransferRequestMessage(TransferRequest request) {
         Properties props = new Properties();
         Session session = Session.getDefaultInstance(props, null);
 
-        // TODO make this message contain useful information
-        String messageText = "Hey there! \n"
-                + "    " + request.getRequestingUser().getEmail() + " has requested that you transfer ownership of "
-                + "your files in the folder  <a href=\"https://gimmeyourfilesplease.appspot.com/#/request/" + request.getId() + "\">" + request.getId() + "</a>.";
+        String messageFormat = "<h1>Hey there, %s!</h1> \n <p><strong>%s</strong> (%s) has requested that you transfer ownership of your files on Google Drive.  "
+                + "<a href=\"https://gimmeyourfilesplease.appspot.com/#/request/%s\">Click Here</a> to view the files they have requested and manage their ownership.</p>" +

```

Comment:
I think "and manage their ownership" should be removed. I know what it means, but it is confusing.

I think a better wording would be "click here to reply to the request"